queries.sql

```diff
@@ -8,3 +8,35 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
```

Comment:
* Kindly verify that the change was made after `species = unspecified` and verify that the species columns went back to the state before the transaction.
 ![image](https://user-images.githubusercontent.com/52497246/207435729-983980bd-510a-4292-af5a-23e563a6981a.png)